Documentation/writing-principles-and-guidelines/spelling-and-typo3-terminology.rst

```diff
@@ -1,111 +1,1006 @@
+        used in the backend to receive data from the FormEngine (content
+        elements and records), and is also part of an extensive API that can be
+        used by extensions to operate on records.
+
+    ..  _db-analyzer---db-compare:
+    -   *DB Analyzer / DB Compare*
+
+        DB Analyzer and DB Compare are tools in TYPO3 that analyze and compare
+        database structures to identify needed changes on the database-level for
+        both for system upgrades and extension integration. Other systems often
+        call this "database migration".
+
+    ..  _db-mounts---mount-points:
+    -   *DB Mounts / Mount Points*
+
+        Mount points allow TYPO3 editors to mount a page (and its subpages) from
+        a different area in the current backend page tree.
+
+    ..  _dbal:
+    -   *DBAL*
+
+        The Database Abstraction Layer (DBAL) is an abstraction layer in TYPO3
+        that facilitates access to various database systems.
+
+*   E
+
+    ..  _enhancer:
+    -   *Enhancer*
+
+        An enhancer is a component that adds additional functionalities or
+        improvements to existing TYPO3 features, most commonly used for
+        "Routing Enhancers" operating on speaking URL fragments.
+
+    ..  _exclude-fields:
+    -   *Exclude Fields*
+
+        Exclude fields are fields that have been configured to allow being
+        hidden in the TYPO3 backend for specific users or user groups. This is
+        done via the permission setup.
+
+    ..  _extbase:
+    -   *Extbase*
+
+        Extbase is a framework for developing extensions in the TYPO3 system
+        based on the Model-View-Controller (MVC) principle. It allows to easily
+        define models with their data fields, that can be managed by editors in
+        the backend. The models can then also be revealed in the frontend with
+        any custom logic, adhering to standards, conventions and API
+        definitions. The Extbase controllers which are part of an Extbase plugin
+        allow to integrate this custom logic in PHP.
+
+    ..  _extension:
+    -   *Extension*
+
+        An extension is an add-on to the TYPO3 system that adds additional
+        functionalities and features. An extension can consist of multiple
+        parts, for example backend modules, frontend plugins, scheduler tasks,
+        console commands, API definitions, frontend styling and much more.
+
+    ..  _extension-builder:
+    -   *Extension Builder*
+
+        The Extension Builder is a backend module in TYPO3 that facilitates the
+        creation of (preferably) extbase extensions. The Extension Builder is a
+        community extension and maintained on its own.
+
+    ..  _extension-manager:
+    -   *Extension Manager*
+
+        The Extension Manager is an interface in the TYPO3 backend used for
+        installing, updating, and managing extensions. It is very important in
+        legacy installations, but in cComposer-based installations it is only
```

Comment:
```suggestion
        legacy installations, but in composer-based installations it is only
```

Reply:
Actually Composer beeing a name / propper noun it should always be capitalized. So Composer-based not composer-based